COLLABORATOR_GUIDE.md

```diff
@@ -255,14 +289,32 @@ When you send your pull request, consider including information about
 whether your change is breaking. If you think your patch can be backported,
 please feel free to include that information in the PR thread.
 
-#### Who is doing the backporting?
-
-The current plan is for commits to cherry pick into a staging branch (e.g. v4.x-staging),
-which can be done by anyone. The preference would be for the individual landing the commit
-on master to backport to staging branches if it is appropriate.
+Several LTS related issue and PR labels have been provided:
+
+* `lts-watch-v4.x` - tells the LTS WG that the issue/PR needs to be considered
+  for landing in the `v4.x-staging` branch.
+* `lts-watch-v0.10` - tells the LTS WG that the issue/PR needs to be considered
+  for landing in the `v0.10-staging` branch.
+* `lts-watch-v0.12` - tells the LTS WG that the issue/PR needs to be considered
+  for landing in the `v0.12-staging` branch.
+* `land-on-v4.x` - tells the release team that the commit should be landed
+  in a future v4.x release
+* `land-on-v0.10` - tells the release team that the commit should be landed
+  in a future v0.10 release
+* `land-on-v0.12` - tells the release team that the commit should be landed
+  in a future v0.12 release
+  
+Any collaborator can attach these labels to any PR/issue. As commits are
```

Comment:
Yeah, I was stewing over that one a bit. Let's see how things shake out in real use then document the practice that emerges.